roles/postgresql-users/tasks/main.yml

```
@@ -21,4 +21,20 @@
   when: postgresql_users is defined and postgresql_users | length > 0
   tags: postgresql_users
 
+- name: Grant roles to users
+  community.postgresql.postgresql_membership:
+    group: "{{ item.role }}"
```

```suggestion
    group: "{{ item.role | default('') }}"
```
If the 'role' is not specified in postgresql_users, the task will fail because it will try to get a non-existent attribute.